src/components/Pivot/Pivot.scss

```diff
@@ -123,6 +123,14 @@
 
     &.is-selected {
       font-weight: $ms-font-weight-semilight;
+
+      .ms-Pivot-count {
+        font-weight: $ms-font-weight-regular;
+      }
+    }
+
+    .ms-Pivot-count {
+      @include margin-left(5px);
```

Comment:
Hmm. Instead of asymetrical margin, could we use a symetrical one? e.g. margin: 0 5px;

This reduces the generated CSS rules and is in practice a good habit to get into.